src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
 // write your code here
+const collection = document.querySelectorAll('.population');
+
+const population = [...collection]
+  .map((item) => Number(item.innerText.split(',').join('')))
+  .filter((item) => item !== null);
```

Comment:
- You convert values to numbers, so why do you check if some of them are `null`?
- And, you don't need to strictly compare with null, you can just check if the value exists. https://developer.mozilla.org/en-US/docs/Glossary/Falsy